meta/markdown-info.md

```diff
@@ -2,7 +2,7 @@
 
 A markdown file is a plain ASCII text file that is meant to be easy to read in plaintext, while also being pretty once rendered.
 
-It is similar to the format that many wiki's have. As an example, you can see [this rendered pattern](https://github.com/paypal/InnerSourcePatterns/blob/master/dedicated-community-leader.md) of ours also [in plain text](https://raw.githubusercontent.com/paypal/InnerSourcePatterns/master/dedicated-community-leader.md). You can do this for any markdown file by clicking the ``Raw`` button when viewing a file online.
+It is similar to the format that many wiki's have. As an example, you can see [this rendered pattern](dedicated-community-leader.md) of ours also [in plain text](https://raw.githubusercontent.com/InnerSourceCommons/InnerSourcePatterns/master/dedicated-community-leader.md). You can do this for any markdown file by clicking the ``Raw`` button when viewing a file online.
```

Comment:
I left this as a non-relative link, as I couldn't see a way to do a raw link relatively (interestingly, it can be done within the github wiki)

Reply:
I would not know of a way either - should we leave it this way or does someone else maybe know?

Reply:
figured it out 😀